packages/core/src/index.ts

```diff
@@ -415,9 +415,9 @@ export {
   parseDomain,
   Synthesizer,
   showMutations,
-  RenderShape,
   RenderInteractive,
   RenderStatic,
+  RenderShape,
```

Comment:
Since this change just moves the `RenderShape` export lower down in the list, could you revert it? (It just adds unnecessary noise to the diff.)

Reply:
Done